lib/jwt/AccessToken.js

```diff
@@ -41,15 +41,15 @@ _.extend(TaskRouterGrant.prototype, {
  *                 assigned to the user
  * @param {string} options.pushCredentialSid - The Push Credentials SID
  */
-function IpMessagingGrant(options) {
+function ChatGrant(options) {
   options = options || {};
   this.serviceSid = options.serviceSid;
   this.endpointId = options.endpointId;
   this.deploymentRoleSid = options.deploymentRoleSid;
   this.pushCredentialSid = options.pushCredentialSid;
 }
 
-_.extend(IpMessagingGrant.prototype, {
+_.extend(ChatGrant.prototype, {
   key: 'ip_messaging',
```

Comment:
That is a top question. Since the IpMessaging Grant still works I assumed it was because of the content of the token, including the key `ip_messaging`. I will check with the chat team to see if a `chat` key will work too.